app/templates/settings/general.hbs

```diff
@@ -39,13 +39,30 @@
 
                 {{#if showUploadLogoModal}}
                     {{gh-fullscreen-modal "upload-image"
-                            model=(hash model=model imageProperty="logo")
+                            model=(hash model=model imageProperty="logo" accept=imgMimeTypes extensions=imgExtensions allowUrlInput=true)
```

Comment:
Is it necessary to set the `accept` and `extensions` properties here when they are the same as the component default values?